apps/user/interfaces/user.interface.ts

```diff
@@ -23,14 +23,35 @@ export interface Organisation {
 }
 
 export interface UsersProfile {
```

Comment:
At least one field should be mandatory. If there is an requirement for having all fields optional, let's discuss and finalize.

Reply:
Resolved.